askama/Cargo.toml

```diff
@@ -37,9 +37,9 @@ mime = []
 mime_guess = []
 
 [dependencies]
-askama_derive = { version = "0.11.0", path = "../askama_derive" }
+askama_derive = { version = "0.11.2", path = "../askama_derive" }
 askama_escape = { version = "0.10", path = "../askama_escape" }
```

Comment:
Explicitly use 0.10.3?

Reply:
Forgot, and I suppose it will pull that in anyway through askama_shared.

The bigger problem actually is that the bump for Askama 0.11.1 is  technically semver-incompatible because we added a const to a trait. While we usually think of our trait implementations as output, it's possible that someone  has actually written a manual implementation which will now break. I figured I'd take the risk, since the alternatives seem worse.